src/mmw/sass/components/_tables.scss

```diff
@@ -45,7 +45,19 @@
   }
 }
 
-.fixed-table-container thead th .th-inner {
-  white-space: normal;
-  line-height: 1.25;
+.fixed-table-container {
+  padding-bottom: 0px !important;
```

Comment:
Setting `data-height="auto"` above adds a large amount of padding, which we have to remove here.